rpcs3/Emu/System.cpp

```diff
@@ -646,8 +693,7 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 			return game_boot_result::invalid_file_or_folder;
 		}
 
-		sys_log.notice("Title: %s", GetTitle());
-		sys_log.notice("Serial: %s", GetTitleID());
+		sys_log.always()("Title: %s, Serial: %s", GetTitle(), GetTitleID());
```

Comment:
I don't really like this change when I read the log :)

Reply:
Seconded. Also forces retooling of any parsing scripts one may be using. Its also just worse to look at.